mlrun/model_monitoring/db/stores/sqldb/sql_store.py

```diff
@@ -303,18 +303,20 @@ def list_model_endpoints(
 
             # Apply filters
             if model:
+                model = model if ":" in model else f"{model}:latest"
                 query = self._filter_values(
                     query=query,
                     model_endpoints_table=model_endpoints_table,
                     key_filter=mm_schemas.EventFieldType.MODEL,
                     filtered_values=[model],
                 )
             if function:
+                function_uri = f"{self.project}/{function}"
```

Comment:
This is not a URI though.

Reply:
True' but in MEP store it looks like this :)

Reply:
Ah, I see that it's already defined as a URI in MM constants 🤦‍♂️